portable/RVDS/ARM_CM4F/port.c

```diff
@@ -243,14 +243,14 @@ __asm void vPortSVCHandler( void )
     PRESERVE8
 
     /* Get the location of the current TCB. */
-    ldr r3, = pxCurrentTCB
+    ldr r3, =pxCurrentTCB
     ldr r1, [ r3 ]
     ldr r0, [ r1 ]
     /* Pop the core registers. */
-    ldmia r0 !, {r4-r11,r14}
+    ldmia r0 !, { r4 - r11, r14 }

```

Comment:
"r4-r11" was correct, is specifies registers r4 to r11, not a subtraction.  Likewise for all similar edits.